### DISCUSSION TITLE
Add ability to change icons

### DISCUSSION THREAD

**Discussion by j10sanders** [samhirtarif/react-audio-recorder#96] [Ideas]
Hey, love the library!  I would like the ability to change the icons.  Specifically, I want to replace the Save icon with something for Uploading (preferably my own SVG).  Is there a way to do this now?

**Comment by samhirtarif** [comment]
Hey @j10sanders - There is currently no way to change the SVGs 😶. Feel free to contribute if you have the time to do so.
Or I will get around to adding this functionality eventually in a week or so because it is a nice idea.

**Reply by josephrmartinez** [reply]
This would be very helpful. The current selection of SVGs is somewhat confusing.
